app/src/main/res/layout/help_without_drawer_activity.xml

```diff
@@ -8,9 +9,15 @@
     android:layout_height="match_parent"
     android:orientation="vertical">
 
-    <include
-      android:id="@+id/help_activity_toolbar"
-      layout="@layout/toolbar" />
+      <androidx.appcompat.widget.Toolbar
+        android:id="@+id/help_activity_toolbar"
```

Comment:
Ditto here (ID should be unique).

Reply:
Renaming the id triggers an avalanche of test fails as show above PTAL